embedded-keycloak/src/test/resources/realm.json

```diff
@@ -0,0 +1,95 @@
+{
+  "realm": "test-realm",
```

Comment:
who is using this file?

Reply:
Still work in progress, going to use it within the tests for using a predefined realm.